tools/ubuntu-distro-json-generate.mjs

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs-extra';
+import shell from 'shelljs';
+
+shell.echo(`Verifying required packages...`);
+
+if (!shell.which(`distro-info`)) {
+  shell.echo('This script requires distro-info, exiting...');
+  shell.exit(2);
+}
+
+if (!shell.which(`sed`)) {
+  shell.echo('This script requires sed, exiting...');
+  shell.exit(2);
+}
+
+if (!shell.which(`jo`)) {
+  shell.echo('This script requires jo, exiting...');
+  shell.exit(2);
+}
```

Comment:
nit: I would see if we can do more of this in Node as I don't think `jo` is a common default package. But this can be handled at a later date.

Reply:
Yeah, we should probably try to do all in J's, so we can do this easily for other Debian bases distros.

Reply:
done both